### DISCUSSION TITLE
How could I get route.description in app.routes or somewhere?

### DISCUSSION THREAD

**Discussion by skk294** [fastapi/fastapi#6580] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@app.get("/",description='test')
async def index():
    return RedirectResponse("/sign_in")

for route in app.routes:
    print(route.name,route.path) # something like route.description?
```

### Description

I can get name,path,method of a route, but i don't know how to get the description.

### Operating System

Linux

### Operating System Details

20.04

### FastAPI Version

0.66.1

### Python Version

3.9.5

**Comment by NoCrEaTiVe** [comment, marked as answer]
@skk294 
Basic routes(OpenApi) have a class Route in which there is no "description" field
You can write like this
```

from fastapi.routing import APIRoute

app = FastApi()

@app.get("/",description='test')
async def index():
    return RedirectResponse("/sign_in")

for route in app.routes:
    if isinstance(route, APIRoute):
        print(route.name, route.path, route.description)
```

**Comment by skk294** [comment]
> @skk294
> Basic routes(OpenApi) have a class Route in which there is no "description" field
> You can write like this
> 
> ```
> 
> from fastapi.routing import APIRoute
> 
> app = FastApi()
> 
> @app.get("/",description='test')
> async def index():
>     return RedirectResponse("/sign_in")
> 
> for route in app.routes:
>     if isinstance(route, APIRoute):
>         print(route.name, route.path, route.description)
> ```

Thanks.(￣▽￣)/
